projects/golang/go/Makefile

```diff
@@ -121,6 +154,14 @@ ifndef GO_SOURCE_VERSION
 	$(error environment variable GO_SOURCE_VERSION, e.g. 1.15, is undefined)
 endif
 
+.PHONY: set-env
```

Comment:
I dont believe this is going to work like you want. Each shell execution in make is a new shell. So even tho this one exports the path, it wont still be in affect by the time rpmbuild is called

Reply:
I would probably jsut simplify this because we know we are builder from the builder base so we could just add this at the top of the makefile

`export PATH:=/go/goGO_SOURCE_VERSION/bin:$(PATH)`

Reply:
✅